### DISCUSSION TITLE
Set a description on custom command

### DISCUSSION THREAD

**Discussion by finnar-bin** [helix-editor/helix#7741] [Q&A] [answered]
How do I set a description on a custom command I added on the space menu? I just find it pretty odd to be empty since I use "space q q" to run ":qa" since I'm pretty used to that in nvim. The command works just fine would just be better in the modal it would have a description.

```
[keys.normal.space]
q = { q = ":qa", w = ":wqa" }
```

![image](https://github.com/helix-editor/helix/assets/28705606/8126231b-988b-4168-a4ce-b9f3ea76d50f)

**Comment by the-mikedavis** [comment, marked as answer]
It's not currently possible, see https://github.com/helix-editor/helix/issues/1752

**Reply by finnar-bin** [reply]
Gotcha, I see there are several PRs attempting to add this. Thanks for answering!
